pkg/exporter/syslogexporter/README.md

```diff
@@ -1,29 +1,46 @@
 # Syslog Exporter
 
-Export/Forward syslog messages to a syslog server
-
 **Stability level**: Experimental
 
-This exporter can forward syslog messages to a third party [syslog server](https://www.rsyslog.com/)
+## About The Exporter
+
+The syslog exporter/forwarder supports sending messages to a remote syslog server.
```

Comment:
I'd just avoid calling it a `forwarder` anywhere, I think that's more confusing than helpful.